contracts/src/v0.8/vrf/VRFV2Wrapper.sol

```diff
@@ -87,24 +91,22 @@ contract VRFV2Wrapper is ConfirmedOwner, TypeAndVersionInterface, VRFConsumerBas
     address callbackAddress;
     uint32 callbackGasLimit;
     uint256 requestGasPrice;
-    int256 requestWeiPerUnitLink;
-    uint256 juelsPaid;
```

Comment:
Any reason these were removed?

Reply:
They were always actually useless on the initial wrapper. It had already been deployed by the time we realized this.